maquetado/setup/scss/tech.scss

```diff
@@ -34,6 +34,7 @@ $tech-container-width: 210px;
 .tech-logo {
   margin: 12px;
   width: $tech-logo-size;
+  height: $tech-logo-size;
```

Comment:
Remember this should be alphabetically ordered. As you added this I guess just width wasnt enought, maybe with a height now you can remove width?

Reply:
Can't remove width because of the Express logo. (because is not a square).